test/e2e/reaper_test.go

```diff
@@ -104,11 +110,13 @@ func createMultiReaper(t *testing.T, ctx context.Context, namespace string, f *f
 	defer f.UndeployAllIngresses(t, "kind-k8ssandra-0", namespace)
 	defer f.UndeployAllIngresses(t, "kind-k8ssandra-1", namespace)
 
+	authEnabled := authEnabled(t, f, ctx, framework.ClusterKey{K8sContext: "kind-k8ssandra-0", NamespacedName: kcKey})
```

Comment:
Same here, it will always be true. In fact I'm not sure we need the `authEnabled` function at all.